android/gradle.properties

```diff
@@ -13,3 +13,4 @@ RNVideo_useExoplayerHls=true
 RNVideo_androidxCoreVersion=1.9.0
 RNVideo_androidxActivityVersion=1.7.0
 RNVideo_buildFromMedia3Source=false
+RNVideo_requiredKotlinVersion=1.9.0
```

Comment:
I hope use 1.8.0. Default version of template that brought us to Kotlin is 1.8.0.
(from react-native@0.73.*)

Reply:
I agree - I have tested and 1.8.0 is working
cc @seyedmostafahasani

Reply:
@YangJonghun @KrzysztofMoch it's applied.